pkg/loadbalancer/experimental/writer.go

```diff
@@ -582,3 +594,43 @@ func (w *Writer) DebugDump(txn statedb.ReadTxn, to io.Writer) {
 
 	tw.Flush()
 }
+
+func isExtLocal(fe *Frontend) bool {
+	switch fe.Type {
+	case loadbalancer.SVCTypeNodePort, loadbalancer.SVCTypeLoadBalancer, loadbalancer.SVCTypeExternalIPs:
+		return fe.service.ExtTrafficPolicy == loadbalancer.SVCTrafficPolicyLocal
+	default:
+		return false
+	}
+}
+
+func isIntLocal(fe *Frontend) bool {
+	/* FIXME if !option.Config.EnableInternalTrafficPolicy {
```

Comment:
OT to your PR - any idea why we have a flag for InternalTrafficPolicy?

Reply:
for feature gating purposes